### ISSUE TITLE
Combo giving warning when select_default() is used

### ISSUE THREAD

**Issue by synchrowisdom** [lawsie/guizero#213]
Hi, first of all I would like to thank you for making this wrapper, it's nice and easy to use and the 'warning' provided in the console are of great help in learning to use it.

I could be wrong but I believe there is a bug in the code, possibly from adjusting a previous situation (https://github.com/lawsie/guizero/issues/206). When .select_default() is used, the Combo performs its work perfectly but it looks like it doesn't return the 'True' flag from its successful operation and this warning is sent to the console:
```
  GUIZERO WARNING  
[Combo] object with options  ['choice1', 'choice2']
Unable to select default option as the Combo is empty
```
But clearly the list isn't empty.

I think I have located the issue in Combo.py:
```
def _set_option_by_index(self, index):
        """
        Sets a single option in the Combo by its index, returning True if it was able too.
        """
        if index < len(self._options) - 1:
            self._selected.set(self._options[index])
**#Missing?==>return True;**
        else:
            return False
```

FYI I am using version 0.5.3.

I hope this helps.

Thanks.

EDIT: Just placing pieces of codes and outputs inside a code block.

**Comment by martinohanlon** [comment]
Yes, this looks like a bug. Thanks for reporting and doing the investigation.

As its relatively minor (i.e. the functionality still works but warning is noted), I will target for the next release rather than doing a special bug fix release for it.

**Comment by martinohanlon** [comment]
The fix is in dev.

**Comment by synchrowisdom** [comment]
Thanks, cheers!

**Comment by martinohanlon** [comment]
Included in 0.5.4
